packages/charts/react-charts-preview/library/src/components/Legends/Legends.tsx

```diff
@@ -8,6 +8,7 @@ import { Overflow, OverflowItem } from '@fluentui/react-overflow';
 import { useFocusableGroup, useArrowNavigationGroup } from '@fluentui/react-tabster';
 import { OverflowMenu } from './OverflowMenu';
 import { tokens } from '@fluentui/react-theme';
+import { mergeClasses } from '@griffel/react';
```

Comment:
move the style merging logic to styles file.

Reply:
done